libs/gi/db/src/Interfaces/BuildTc.ts

```diff
@@ -17,6 +18,7 @@ export type BuildTcArtifactSlot = {
 export type BuildTc = {
   name: string
   description: string
+  character?: Omit<ICharacter, 'key'>
```

Comment:
No visualization of override here 
![image](https://github.com/frzyc/genshin-optimizer/assets/36019388/3a63fde6-1361-4a9e-b320-87d16fbf39f8)

Reply:
![Screenshot 2024-05-22 013424](https://github.com/frzyc/genshin-optimizer/assets/1754901/555e4c5f-7e29-40b9-823d-88226e3e175d)